frontend/packages/dev-console/src/components/import/advanced/AdvancedSection.tsx

```diff
@@ -46,35 +46,83 @@ const List: React.FC<AdvancedSectionProps> = ({ appResources, values }) => {
       Footer={Footer}
     >
       <ProgressiveListItem name={t('devconsole~Health checks')}>
-        <HealthChecks title={t('devconsole~Health checks')} resourceType={values.resources} />
+        <div
+          style={
+            visibleItems[0] === 'Health checks'
+              ? { marginTop: 'var(--pf-global--spacer--lg)' }
+              : { marginTop: 'var(--pf-global--spacer--2xl)' }
+          }
+        >
+          <HealthChecks title={t('devconsole~Health checks')} resourceType={values.resources} />
```

Comment:
Please find a more generic way to solve this spacing instead of checking for the first items and adding different margins here.

We can check out this tomorrow together if you want.

Reply:
Maybe by adding spacing above all elements and then using a PF `Flex` grid around all items.